device.mk

```diff
@@ -205,4 +205,26 @@ PRODUCT_PROPERTY_OVERRIDES += \
 #
 PRODUCT_DEFAULT_PROPERTY_OVERRIDES += \
     persist.sys.usb.config=mtp \
```

Comment:
Dave, Thanks for you suggestion. After I update init.shinano.usb.rc, the problem is solve. Now usb can be detected by host, adb can found the device and mass_storage also does work properly.